opm/autodiff/NonlinearSolver_impl.hpp

```diff
@@ -103,11 +103,11 @@ namespace Opm
             }
             linIters += report.linear_iterations;
             ++iteration;
-        } while ( (!converged && (iteration <= maxIter())) || (minIter() > iteration));
+        } while ( (!converged && (iteration <= maxIter())) || (iteration <= minIter()));
```

Comment:
I do notice an error though. The assert above must be removed, since now (due to the minIter again) we may have convergence from the model, yet still it had to actually do a solve (since minIter demanded it). I'll remove that.